src/pages/academy/game/Game.tsx

```diff
@@ -50,9 +51,82 @@ function Game() {
     }
   }, [session, achievements, goals]);
 
+  // Logic for the fullscreen button to toggle fullscreen.
+  // Entering fullscreen will also lock the screen orientation to landscape.
+  const [isFullScreen, setIsFullScreen] = React.useState(false);
+  const toggleFullScreen = () => {
```

Comment:
For this function, as we are using React, using the `ref` here would be more appropriate instead of accessing the element directly from the browser through `document`,